mtags/src/main/scala-3/scala/meta/internal/pc/PcCollector.scala

```diff
@@ -99,68 +100,103 @@ abstract class PcCollector[T](driver: InteractiveDriver, params: OffsetParams):
+          if pos.span.start > arg.span.start && pos.span.end < arg.span.point + realName.length
+          then
+            appl.symbol.paramSymss.flatten.find(_.name == arg.name).map { s =>
+              // if it's a case class we need to look for parameters also
+              if caseClassSynthetics(s.owner.name) && s.owner
+                  .is(Flags.Synthetic)
+              then
+                Set(
+                  s,
+                  s.owner.owner.companion.info.member(s.name).symbol,
+                  s.owner.owner.info.member(s.name).symbol,
+                )
+                  .filter(_ != NoSymbol)
+              else Set(s)
+            }
+          else None
+          end if
+        /* all definitions:
+         * def fo@@o = ???
+         * class Fo@@o = ???
+         * etc.
+         */
+        case (df: NamedDefTree) :: _ if df.nameSpan.contains(pos.span) =>
+          Some(symbolAlternatives(df.symbol))
+        /**
+         * For traversing annotations:
+         * @JsonNo@@tification("")
+         * def params() = ???
+         */
+        case (df: MemberDef) :: _ if df.span.contains(pos.span) =>
+          val annotTree = df.mods.annotations.find { t =>
+            t.span.contains(pos.span)
           }
-        else None
-        end if
-      /* all definitions:
-       * def fo@@o = ???
-       * class Fo@@o = ???
-       * etc.
-       */
-      case (df: NamedDefTree) :: _ if df.nameSpan.contains(pos.span) =>
-        Some(symbolAlternatives(df.symbol))
-      /**
-       * For traversing annotations:
-       * @JsonNo@@tification("")
-       * def params() = ???
-       */
-      case (df: MemberDef) :: _ if df.span.contains(pos.span) =>
-        val annotTree = df.mods.annotations.find { t =>
-          t.span.contains(pos.span)
-        }
-        collectTrees(annotTree).flatMap { t =>
-          soughtSymbols(
-            Interactive.pathTo(t, pos.span)
-          )
-        }.headOption
+          collectTrees(annotTree).flatMap { t =>
+            soughtSymbols(
+              Interactive.pathTo(t, pos.span)
+            )
+          }.headOption
 
-      /* Import selectors:
-       * import scala.util.Tr@@y
-       */
-      case (imp: Import) :: _ if imp.span.contains(pos.span) =>
-        imp.selector(pos.span).map(symbolAlternatives)
+        /* Import selectors:
+         * import scala.util.Tr@@y
+         */
+        case (imp: Import) :: _ if imp.span.contains(pos.span) =>
+          imp.selector(pos.span).map(symbolAlternatives)
+
+        case _ =>
+          None
 
-      case _ =>
-        None
+      sought match
+        case Some(value) => sought
+        case None =>
+          // Fallback check for extension method parameter
+          def collectExtMethods(
+              acc: Set[(Symbol, SourcePosition)],
+              tree: untpd.Tree,
+          ) =
+            tree match
+              case ExtMethods(params :: _, _) =>
+                acc ++ params
```

Comment:
params will be both type params and term params?

Reply:
You're right, I've changed it so it should work also when we have type params